user/tests/app/input_pinmode/input_pinmode.cpp

```diff
@@ -14,7 +14,7 @@ InterruptMode intType = CHANGE;
 // We will set most pins to INPUT_x modes, but only test the first 7
 // since PM_INT takes up one of the 8, and after that RISING and CHANGE pending interrupt tests will fail.
 //
-#if (PLATFORM_ID == PLATFORM_P2)
+#if (PLATFORM_ID == PLATFORM_P2) || (PLATFORM_ID == PLATFORM_TRACKERM)
```

Comment:
We should limit TRACKERM to use only D8 & D9.  Unfortunately this test starts always at D0, so it would require a PIN_MAX => START_PIN / END_PIN change.

Reply:
There are multiple ways these pins can be added but looks like adding START_PIN is probably the simplest way.